src/test/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidderTest.java

```diff
@@ -322,6 +323,141 @@ public void makeBidsWithUnparsableBidResponseShouldReturnError() {
 
     }
 
+    @Test
+    public void makeHttpRequestsImpExtComparison() {
+        // given
+        final String placementId = "6821";
+        final Imp givenImp = givenImpWithExt(imp -> imp
+                        .id("custom_imp_id")
+                        .tagid("custom_imp_tagid")
+                        .secure(1)
+                        .banner(Banner.builder()
+                                .w(728)
+                                .h(90)
+                                .format(singletonList(Format.builder()
+                                        .w(728)
+                                        .h(90)
+                                        .build())).build()
```

Comment:
Extract deeply nested sequences like this to locals, to reduce nesting. Also, please check for similar occurences.

Reply:
@And1sS fixed